server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_4_alter_regulation_riskdetect_rule.sql

```diff
@@ -0,0 +1,4 @@
+--
+-- Alter column `value_json` size from varchar(2048) to varchar(10240) in `regulation_riskdetect_rule` table
+--
+alter table `regulation_riskdetect_rule` modify column `value_json` varchar(10240) default null comment 'json of rule value';
```

Comment:
4.3.2.4 is occupied, 4.3.2.11 is available

Reply:
got it